config/locales/home.ko.yml

```diff
@@ -22,222 +22,216 @@
+  calc_diversity: 견해 
+  users: 사용자
+  diversity_score: 견해 점수
+  calc_diversity_sub: 점수를 말해줘!
+  clear: 지우기
+  calc_it: 다양성 점수 계산
+  ds_emails_lbl: 팀멤버 이메일 입력
+  dates: 날짜
+  open_date: 개시 날짜
+  start_date: 시작 날짜
+  close_date: 종료 날짜
+  completion_date: 완료 날짜
+  individual: 개인
+  type: 종류
+  expired: 기한 완료
+  bingo_game: 빙고 게임!
+  experience: 그룹 경험
+  assignment: 과제
+  other_info: 기타 정보
+  start_dow: DOW 시작
+  end_dow: DoW 끝
+  end_date: 마감 날짜
+  task_view: 할 일 보기
+  month_view: 월별 보기
+  week_view: 주간 보기
+  activities: 활동
+  consent_form: 동의서 양식
+  actions: 실행
+  activate: 활성화
+  back: 뒤로
+  next: 다음
+  note: 메모
+  show: 보여주기
+  edit: 편집하기
+  save: 저장하기
+  add: 추가하기
+  submit: 제출하기
   #toolbar
-  profile: Profile
-  c_panel: Admin&hellip;
-  reports: Reports
-  administration: Administration
-  resp_summ: Response Summary
-  resp_data: Response Data
-  coming_soon: Coming soon!
-  logout: Logout
-  login: login
-  none_selected: None selected
-  not_available: Not Available
+  profile: 개요
+  c_panel: 관리 패널&hellip;
+  reports: 보고서
+  administration: 관리
+  resp_summ: 답변 요약
+  resp_data: 답변 날짜
+  coming_soon: 개봉 박두!
+  logout: 로그아웃
+  login: 로그인
+  none_selected: 선택 안함
+  not_available: 사용 불가
   #demo materials
-  demo_title: '%{orig} (DEMO)'
-  demo: Demo
-  demo_group: SuperStars
-  demo_project: Research Paper
+  demo_title: '%{orig} (데모)'
+  demo: 시범
+  demo_group: 수퍼스타들
+  demo_project: 연구보고서
   demo_project_description: >
-    This project involves researching the topic decided in your class
-    and working with your team to author a paper thoroughly detailing
-    all you've learned on the topic.
-  demo_course_name: Advanced Collaborative Research
+    이 프로젝트는 수업에서 결정된 주제를 연구하고, 팀과 협력하여 
+    배운 내용을 자세히 설명하는 논문을 작성하는 것을 목표로 합니다.
+  demo_course_name: 고급 협력 연구
   #admin functions
-  about: About
-  back_to_course: Back to course
-  back_to_courses: Back to the list of courses
-  support: Support
+  about: 소개
+  back_to_course: 코스로 돌아가기
+  back_to_courses: 코스 목록으로 돌아가기
+  support: 지원
   avail_body: >
-    The %{activity_name} is now available for you on %{site_link}. Please
-    %{login_link} at your earliest convenience to have a look and complete
-    it.
+    %{activity_name} 활동이 %{site_link}에서 사용 가능합니다. 가능한 빨리 
+    %{login_link}에 로그인하여 확인하고 완료해 주세요.
   course_invite_body: >
-    (1){You've been invited to participate in the following course:};
-    (2-inf){You've been invited to participate in the following {{count}} courses:}
-  course_invite_close: Please log in to %{site_link} accept or decline this enrollment.
+    (1){다음 수업에 참여하도록 초대되었습니다:};
+    (2-inf){다음 {{count}}개의 수업에 참여하도록 초대되었습니다:}
+  course_invite_close: %{site_link}에 로그인하여 이 등록을 수락하거나 거부하세요.
   reminder_body: >
-    This email is to serve as a friendly reminder that you have an open
-    task waiting for you at this time. Please login to %{site_link} at your
-    earliest convenience to complete what is available.
+    이 이메일은 현재 대기 중인 할 일이 있음을 상기시켜 드리기 위해 발송되었습니다. 
```

Comment:
I cannot tell what this is attached to and it appears to be a duplication of the item below. Does this belong here?